packages/commonwealth/server/routes/starCommunity.ts

```diff
@@ -32,7 +33,7 @@ const starCommunity = async (
     }
     return res.json({ status: 'Success' });
   } else {
-    return next(new Error(Errors.NoStarValue));
+    return next(new AppError(Errors.NoStarValue));
```

Comment:
When does this get hit? Is this invalid DB state, or typical?

Reply:
reworked this whole route, it's easiest.